blimp-live-widgets/src/components/GoogleCalendar/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Calendar, momentLocalizer } from 'react-big-calendar'
+import moment from 'moment'
+import '!style-loader!css-loader!react-big-calendar/lib/css/react-big-calendar.css'
+
+const localizer = momentLocalizer(moment)
+
+export default class GoogleCalendar extends Component {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      events: []
+    };
+  }
+
+  signIn = () => {
+    self = this;
+    gapi.auth2.getAuthInstance().signIn().then(function() {
+      self.events();
+    });
+  };
+  signOut = () => {
+    this.setState({ events: [] })
+    gapi.auth2.getAuthInstance().signOut();
+    gapi.auth2.getAuthInstance().disconnect(); //cannot automatically sign in
+  };
+  calendar = () => {
+    var request = gapi.client.calendar.calendarList.list({
+    }).then(function(response) {
+      console.log(response.result.items);
+    });
+  };
+
+  selectEvent = (event) => {
+    console.log(event);
+  };
+
+  events = () => {
+    var self = this;
+    var request = gapi.client.calendar.events.list({
+      'calendarId': "primary",
+      'timeMin': (new Date()).toISOString(),
+      'showDeleted': false,
+      'singleEvents': true,
+      'maxResults': 10,
+      'orderBy': 'startTime'
+    }).then(function(response) {
+      var events = response.result.items;
+      var event_list = []
+
+      for (var i=0; i < events.length; i++) {
+        event_list.push({
+          id: events[i].id,
+          title: events[i].summary,
+          start: moment(events[i].start.dateTime, 'YYYY-MM-DD').toDate(),
+          end: moment(events[i].end.dateTime, 'YYYY-MM-DD').toDate()
+        });
+      }
+      self.setState({
+        events: event_list
+      });
+    });
+  };
+
+  componentDidMount() {
+    var self = this;
+    gapi.load('client:auth2', function() {
+      gapi.client.init({
+        apiKey: 'AIzaSyAgcRDOmfb0eEv2gWKEFW-_hR2UxwHTkfI',
```

Comment:
can u add a comment that we should store this securely in the future - same w/ client ID